Packs/ShiftLeft/Integrations/shiftleft/shiftleft.py

```diff
@@ -0,0 +1,338 @@
+    findings = response.get("findings", {})
+    scan = response.get("scan", [])
+    if scan:
+        scan = [scan]
+    # Exclude SDL
+    filtered_findings = [
+        f
+        for f in findings
+        if f.get("owasp_category") not in ("a3-sensitive-data-exposure")
+    ]
+    reachable_oss_found = False
+    for f in filtered_findings:
+        f["sink_method"] = ""
+        f["source_method"] = ""
+        f["file_locations"] = ""
+        f["reachable_oss"] = ""
+        f["cves"] = ""
+        if f.get("details"):
+            details = f.get("details")
+            f["sink_method"] = details.get("sink_method", "")
+            f["source_method"] = details.get("source_method", "")
+            f["file_locations"] = "\n".join(details.get("file_locations", [])) + "\n"
+        if f.get("related_findings"):
+            oss_vuln = f.get("related_findings").get("oss_vuln")
+            if oss_vuln:
+                reachable_oss_found = True
+                f["reachable_oss"] = len(oss_vuln)
+                cve_tags = []
+                for ov in oss_vuln:
+                    cve_tags += [
+                        t.get("value")
+                        for t in ov.get("tags", [])
+                        if t.get("key") in ("cve")
+                    ]
+                f["cves"] = "\n".join(cve_tags)
+    if response:
+        markdown = "### Scan Info\n"
+        markdown += tableToMarkdown(
+            "",
+            scan,
+            headers=[
+                "id",
+                "app",
+                "language",
+                "version",
+                "started_at",
+                "completed_at",
+                "number_of_expressions",
+            ],
+        )
+        markdown += "\n### Recommendation\n"
+        if reachable_oss_found:
+            markdown += "\nFindings with attacker-reachable opensource vulnerabilities requires your attention!"
+        else:
+            markdown += "\nNo findings with attacker-reachable opensource vulnerabilities was found."
+        markdown += f"\n### ShiftLeft Findings ({len(filtered_findings)})\n"
+        markdown += tableToMarkdown(
+            "",
+            filtered_findings,
+            headers=[
+                "id",
+                "severity",
+                "title",
+                "file_locations",
+                "cves",
+            ],
+        )
+        return CommandResults(
+            readable_output=markdown,
+            outputs_prefix="ShiftLeft",
+            outputs_key_field="scan.id",
+            outputs=response,
+        )
+    else:
+        return CommandResults(
+            outputs_prefix="ShiftLeft",
+            outputs_key_field="",
+            outputs={},
+        )
+
+
+""" MAIN FUNCTION """
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    org_id = demisto.params().get("org_id")
+    access_token = demisto.params().get("access_token")
```

Comment:
Please save the params in a variable once to prevent loading the server configuration each time, it is more efficient. For example: (change in other places too).
```suggestion
     params = demisto.params()
    org_id = params.get("org_id")
    access_token = params.get("access_token")
```